src/NodaTime/CalendarSystem.cs

```diff
@@ -232,10 +236,31 @@ public static CalendarSystem ForId(string id)
         /// <returns>A Hebrew calendar system for the given month numbering.</returns>
         public static CalendarSystem GetHebrewCalendar(HebrewMonthNumbering monthNumbering)
         {
-            Preconditions.CheckArgumentRange(nameof(monthNumbering), (int) monthNumbering, 1, 2);
-            return HebrewCalendars.ByMonthNumbering[((int) monthNumbering) - 1];
+            Preconditions.CheckArgumentRange(nameof(monthNumbering), (int)monthNumbering, 1, 2);
```

Comment:
Can you undo all this source reformatting? (I'll fix the whitespace of line 173 which is currently wrong.)
Basically, it would be good to make the changes *just* the ones to do with the Wondrous calendar.